README.md

```diff
@@ -1160,11 +1223,8 @@ Create the `run_multimer_jobs_SLURM.sh` script and place the following code in i
 #Adjust this depending on the node
 #SBATCH --ntasks=8
 #SBATCH --mem=64000
-
-module load Anaconda3 
-module load CUDA/11.8.0
-module load cuDNN/8.7.0.84-CUDA-11.8.0
-source activate AlphaPulldown
+eval "$(conda shell.bash hook)"
+conda activate AlphaPulldown
```

Comment:
replace with source activate AlphaPulldown which always work while the other one works only depending on how the conda was installed?